cabal-ghc921.project

```diff
@@ -36,11 +60,38 @@ package *
 
 write-ghc-environment-files: never
 
-index-state: 2021-12-29T12:30:08Z
+index-state: 2022-01-03T18:45:00Z
 
 constraints:
   -- These plugins doesn't work on GHC92 yet
-  haskell-language-server +ignore-plugins-ghc-bounds -brittany -class -fourmolu -splice -stylishhaskell -tactic -refineImports -callhierarchy -retrie
+  haskell-language-server
+    +ignore-plugins-ghc-bounds
+    -alternateNumberFormat
+    -brittany
+    -callhierarchy
+    -class
+    -eval
+    -floskell
+    -fourmolu
+    -haddockComments
+    -hlint
+    -moduleName
+    -ormolu
+    -qualifyImportedNames
+    -refineImports
+    -retrie
+    -splice
+    -stylishhaskell
+    -tactic,
+  ghc-lib-parser ^>= 9.2,
```

Comment:
would be nice to have comments about these so we know when we can get rid of them

Reply:
about which, the flags or the constraints?

Reply:
The constraints, the flags are clear enough.

Reply:
I don't recall, sadly